base/client.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: &app cockroachdb-client
+  annotations:
+    "app.uw.systems/description": "Used to connect and query the cockroachdb databases."
+    "app.uw.systems/repos.cockroachdb-manifests": "https://github.com/utilitywarehouse/cockroachdb-manifests"
+    "app.uw.systems/tier": "tier_4"
```

Comment:
@matthewhughes-uw Do you have any view on how to make this "not bad"?

Annotation is required by the validator, but specifying "tier" on base manifests doesn't mean anything.

Reply:
> but specifying "tier" on base manifests doesn't mean anything
what do you mean? teams are still able to overwrite it via patches if they want to, right?

Reply:
Yes, and if thats how we want to play it - we need to let them know they NEED to overwrite. Otherwise it will lead to surprising behaviour we'd want to avoid.

Reply:
I think this is only the client pod used for connecting to the CRDB easily. It's not the DB itself.
I think it makes sense to be only tier 4 everywhere. Can't think of a case where it could be more than that.

Reply:
FYI, for the actual DB [tier_1](https://github.com/utilitywarehouse/cockroachdb-manifests/blob/master/base/statefulset.yaml#L9) is used, but I think that's also a good default.